source/_integrations/input_boolean.markdown

```diff
@@ -46,7 +46,6 @@ Input booleans can also be configured via {% term "`configuration.yaml`" %} file
         description: Initial value when Home Assistant starts.
         required: false
         type: boolean
-        default: false
```

Comment:
Unfortunately, this change isn't correct. See also this code snippet below.

![CleanShot 2024-11-28 at 08 10 01@2x](https://github.com/user-attachments/assets/b49e5e34-d083-4f5c-b547-f6ac1fcb0206)

Nevertheless, thanks for being willing to contribute 👍 

../Frenck

Reply:
According to Docs, there are 3 options:
`initial`= true -> the value is set to ON on HA start
`initial`= false -> the value is set to OFF on HA start
`initial` not defined -> the value is restored

Here "default = false" make a wrong impression that the value = OFF by default on HA restart:
![image](https://github.com/user-attachments/assets/1ca40cb4-2ebe-4d4c-ad8e-29001015062e)
But in fact the value is restored because `initial` is not set.
What I am missing then if I are saying I am wrong?

Reply:
No you are right, on the state restore there is another behavior defined:

![CleanShot 2024-11-28 at 08 31 30@2x](https://github.com/user-attachments/assets/a352d4f9-6b79-4a27-bf0b-b13857d33528)


So your adjustment is actually correct due to later defined behavior (beyond the defaults).

The difference would be:

- `initial`= true -> the value is set to ON on HA start
- `initial`= false -> the value is set to OFF on HA start
- `initial` not defined -> the value is restored, _**if there is no value to be restored, it is set to `OFF` on HA start**_

Reply:
Boolean types should always set the default in our documentation.

I would suggest setting it to `default: "Restored from previous"` or something similar.

../Frenck